src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs

```diff
@@ -116,14 +118,18 @@ public class SimpleTestClass : ITestClass
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
                 @"""MyDateTimeOffset"" : ""2019-01-30T12:01:02.0000000+01:00""," +
                 @"""MyEnum"" : 2," + // int by default
-                //@"""MyStringToStringKeyValuePair"" : {""Key"" : ""myKey"", ""Value"" : ""myValue""}," +
+                @"""MyInt64Enum"" : -9223372036854775808," +
+                @"""MyUInt64Enum"" : 18446744073709551615," +
+                 //@"""MyStringToStringKeyValuePair"" : {""Key"" : ""myKey"", ""Value"" : ""myValue""}," +
```

Comment:
Do you know why this was commented out (originally)?

Reply:
by my understanding Steve refactored them

Reply:
@steveharter - what's the plan for these?

Reply:
Those can now be commented back in. The original KeyValue code was removed and replaced with a converter.

Also FYI there were also some tests that assumed "{""a"":b}" implies Key:a and Value:b which was removed\changed to require explicit "Key" and "Value" property names.